src/DonorDashboards/resources/views/donordashboard.php

```diff
@@ -5,11 +5,13 @@
  * @since 2.10.0
  */
 
+use Give\Helpers\Form\Template\Utils\Frontend as FrontendFormTemplateUtils;
 use Give\Views\IframeContentView;
 
+$formId     = FrontendFormTemplateUtils::getFormId();
 $iframeView = new IframeContentView();
 
 echo $iframeView
-    ->setTitle(esc_html__('Donor Dashboard', 'give'))
+    ->setTitle(esc_html__('Donor Dashboard', 'give'))->setFormId($formId)
```

Comment:
@glaubersilva Set form id for `the_title` filter will be wrong because In Donor Dashboard is a page.